boards/posix/native_posix/doc/board.rst

```diff
@@ -573,9 +555,9 @@ installed in the system.
 Legacy shell support
```

Comment:
we probably should not be documenting the legacy and instead just be talking about the new shell if it applies here.

Reply:
This section only applies to the legacy shell and the native_posix console driver that was used with it. For the new shell there is only the UART backend.

I was thinking that when the legacy shell is removed from Zephyr, this native_posix driver and this section would be removed.